.env

```diff
@@ -0,0 +1,5 @@
+DEV_PRIVATE_KEY="ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
Where are these keys coming from? Should we provide defaults at all or have people enter their own?

Assuming the key isn't used for real deployments, this may be fine. But we also don't want people accidentally using this key on mainnet. Even using it on testnets may cause confusion.

Reply:
doh - this branch needed a rebase over [#br-config](https://github.com/Geo-Web-Project/core-contracts/pull/35) since that branch implemented `.env` - ive done that specifically on this branch for now to get rid of the `.env` file for now.